data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
@ranisalt you're mistaken, it's a fair comparation, valve distribute their server same as tfs, you can configure it to have your own server, ofc they don't provide source code for engine, whici means you can have a running dota server up running and connect throught official client, just like counter-strike 1.6 if you remember, then user has to validade their scripts as well, my point was they trust pretty much in their lua api, they provide their host with runnign server to your scripting, that what i meant, anyway it gonne too off topic now, the problem never was "crashable" but without onMoveItem event the store will not work properly, it's my problem with that. tho im ok with a new event to onMoveStoreInbox, as i said before user should be able to change default behaviour, whats the point of engine rely on scripting? that's awful.